src/app/models/Region.ts

```diff
@@ -47,6 +48,8 @@ export interface IRegion extends HasAllUsers, HasDescription {
  * A region model.
  */
 export class Region extends AbstractModel<IRegion> implements IRegion {
+  public static defaultProjectId = 1;
```

Comment:
100% rejected

Reply:
2 immediate alternatives i see that would continue down the road ive set:

1. Store this value in the config sent by the server (ie. `defaultProject: 1`)
2. During bootstrap, make a request for projects if `hideProjects` is set, and extract the project id for the only project (assuming that is true)

Reply:
plan outlined in comment above, but, yeah, option 2 essentially - except not assuming there is a single project